### PR TITLE
Make files_uploader::FilesUploadSummary public

### PR DESCRIPTION
**Pull request by happybeing**
## Description
Enable users of module `autonomi` to import `files_uploader::FilesUploadSummary`.

reviewpad:summary

### CODE REVIEW

**Comment by RolandSherwin** [comment]
Hey @happybeing, ah yeah, it slipped through the cracks and clippy didn't complain :( 

Anyway, I think you might have to update the commit message to follow the convention as mentioned here https://github.com/conventional-changelog/commitlint/#what-is-commitlint

You can reword it to `chore(cli): make FilesUploadSummary public`

**Review by RolandSherwin — APPROVED** [review]

**Comment by happybeing** [comment]
Oops, thanks. I don't do this often enough and tend to forget that. Fingers crossed now.

**Comment by RolandSherwin** [comment]
Ah, you might have to change the capitalization there `Make -> make`
c.c @happybeing

**Comment by happybeing** [comment]
Oh dear sorry. Until tomorrow!
